ai/orchestrators/models-config.mdx

```diff
@@ -31,7 +31,12 @@ currently **recommended** models and their respective prices.
     "pipeline": "upscale",
     "model_id": "stabilityai/stable-diffusion-x4-upscaler",
     "price_per_unit": 4768371,
-  }
+  },
+  {
+    "pipeline": "audio-to-text",
+    "model_id": "openai/whisper-large-v3",
+    "price_per_unit": 12882811,
```

Comment:
@rickstaa Curious about your thoughts on pricing. The audio-to-text pipeline uses milliseconds as the unit (second * 1000)

Reply:
@eliteprox I like the pricing per millisecond 👍🏻. The pricing seems a bit low, but that's fine since we can let the market take effect. With the current pricing, it would cost `12882811*1000*60*10**-18*3205.48 = $0.0025` per minute of audio, whereas [OpenAI charges](https://openai.com/api/pricing/) `$0.006`, making us very competitive.